docs/install/host-hardware-requirements.md

```diff
@@ -14,9 +18,10 @@ If you need help requesting a server environment, please see the [Sample IT Depa
 
 **Please select your hardware environment:**
 
-- [Production Server](#production-server)
-- [Staging Server](#staging-server)
-- [Personal Computer (Demo or Local)](#personal-computer-demo-or-local)
+- [Hardware Requirements](#hardware-requirements)
+  - [Production Server](#production-server)
+  - [Staging Server](#staging-server)
+  - [Personal Computer (Demo or Local)](#personal-computer-demo-or-local)
```

Comment:
Not sure how this change got introduced, since all I did was add the disclaimer. I can revert to the way the content is on the public version of the site, aka the deleted lines.